vizro-ai/docs/pages/user-guides/add-generated-chart-usecase.md

```diff
@@ -103,4 +105,56 @@ This object must then be passed to the `figure` argument of the Vizro [Graph](ht
     [VizroDashboard]: ../../assets/user_guides/chart_into_dashboard_large.gif
 
 
+### Direct use of `fig` object
+
+Additionally, we can assign the output of Vizro-AI `plot()` method directly to the fig variable, enabling its reuse in the `vm.Graph.figure` argument.
+This method offers streamlined efficiency, eliminating the need for code copying. However, it is important to note that each dashboard run triggers an API call to the OpenAI, potentially escalating costs.
```

Comment:
it might be a bit misleading here if we say each dashboard run triggers an API call to openai. In theory if fig is returned with `fig = vizro_ai.plot(df, xx)`, fig is returned as fig object, and only restarting dashboard won't trigger cost.

added suggestion as below